docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/loading-transformed-data/snowflake-loader/index.md

```diff
@@ -10,59 +10,25 @@ import CodeBlock from '@theme/CodeBlock';
 ```
 
-It is possible to run Snowflake Loader on both AWS and GCP.
-
-### Running on AWS
-
-There are two ways to set up the necessary Snowflake resources and run the loader on AWS:
-
-- using our dedicated open source Terraform modules to create the resources and deploy the loader on EC2
-- creating the resources and running the application manually.
-
-We recommend the first way.
-
-### Running on GCP
-
-At the moment, terraform modules for deploying Snowflake Loader on GCP aren't implemented. Therefore necessary Snowflake resources needs to be created and application needs to be deployed manually.
-
-## Using the Terraform modules (for AWS)
-
-### Requirements
-
-- Terraform >= 1.0.0
-- [`terraform-aws-snowflake-loader-ec2` module](https://registry.terraform.io/modules/snowplow-devops/snowflake-loader-ec2/aws/latest)
-
-### Usage
-
-If you have an existing pipeline you should be able to attach the loader to an existing Enriched Kinesis stream following the documentation found [here](https://github.com/snowplow-devops/terraform-aws-snowflake-loader-ec2#usage).
-
-We also have full pipeline deployment examples [here](https://github.com/snowplow/quickstart-examples).
-
-## Manual setup and deployment
+It is possible to run Snowflake Loader on AWS, GCP and Azure.
 
 ### Setting up Snowflake
 
-The following resources need to be created:
+SQL script in [here](/docs/getting-started-on-snowplow-open-source/quick-start/?warehouse=snowflake#prepare-the-destination) can be used to create most of the necessary Snowflake resources.
 
-- Snowflake loader [user](https://docs.snowflake.com/en/sql-reference/sql/create-user.html)
-- Snowflake loader [role](https://docs.snowflake.com/en/sql-reference/sql/create-role.html)
-- Snowflake [warehouse](https://docs.snowflake.com/en/sql-reference/sql/create-warehouse.html)
-- Snowflake [database](https://docs.snowflake.com/en/sql-reference/sql/create-database.html)
-- Snowflake [storage integration](https://docs.snowflake.com/en/sql-reference/sql/create-storage-integration.html)
-- Snowflake [file format](https://docs.snowflake.com/en/sql-reference/sql/create-file-format.html)
-- Snowflake [stage](https://docs.snowflake.com/en/sql-reference/sql/create-stage.html) to load transformed events.
+Also, there are two different load auth methods with Snowflake Loader. With `TempCreds` method, no additional Snowflake resources are needed. With `NoCreds` method, however, Loader needs Snowflake stage therefore it needs to be created prior as well.
 
-<AutoSchemaCreation name="Snowflake" grantDocs="https://docs.snowflake.com/en/sql-reference/sql/grant-privilege" />
+To create a Snowflake stage, you need a Snowflake database, Snowflake schema, Snowflake storage integration, Snowflake file format, and the blob storage (S3, GCS and Azure) path to the transformed events bucket.
```

Comment:
My understanding is that this setup is optional, so we should mark it as such. @istreeter / @jbeemster could you confirm?